File: ExperimentsData/Models/DAO/AttributeEntity.cs
using System;
using ExperimentsData.Models.Enums;

namespace ExperimentsData.Models.DAO
{
    public class AttributeEntity : BaseEntity
    {
        
        public string Value { get; set; }
        public string NormalizedValue { get; set; }
        public int order { get; set; }
        public string name { get; set; }
        public Guid SampleEntityId { get; set; }
        public SampleEntity SampleEntity { get; set; }
    }
}

File: commons.module/src/main/java/br/com/lukkascost/commons/module/models/entities/AttributeEntity.java
package br.com.lukkascost.commons.module.models.entities;

import lombok.*;

import javax.persistence.*;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Table(name = "attributes")
@Entity
public class AttributeEntity extends BaseEntity{

    @Column(name = "value")
    private String value ;

    @Column(name = "order_field")
    private int order ;

    @Column(name = "name")
    private String name ;

    @ManyToOne(fetch = FetchType.LAZY, optional = false)
    @JoinColumn(name = "sample_id", nullable = false)
    private SampleEntity sample;

}
